assets/js/src/core.js

```diff
@@ -0,0 +1,40 @@
+function getNewGameUri() {
```

Comment:
GET seems simpler, and I think it is the appropriate method for ajax requests. In most cases, I'm transferring an integer, so POST seemed especially unnecessary. I don't fully understand the scenarios of using one over the other, so I'm open to changing it.